CurrentIG_editorial review for Elasticsearch & Kibana unattended inst… (#3772)
wazuh/wazuh-documentation: source/installation-guide/open-distro/distributed-deployment/unattended/unattended-elasticsearch-cluster-installation.rst

Elasticsearch & Kibana unattended installation
==============================================

> You can install Open Distro for Elasticsearch and Open Distro for Kibana using an automated script. This script performs a health check to verify that the system has enough resources to achieve optimal performance. For the installation, the script uses the Search Guard offline TLS tool to create the certificates.

Reviewer: ```suggestion
You can install Open Distro for Elasticsearch and Open Distro for Kibana using an automated script. This script performs a health check to verify that the system has enough resources to achieve optimal performance. The script uses the Search Guard offline TLS tool to create the certificates.
```

      The highlighted lines indicate the values that must be replaced in the ``config.yml``. These values are: 

        - ``<elastic_cluster>``: Name of the cluster 
        - ``<master_node_x>``: Name of the node ``X``
        - ``<elasticsearch_ip_nodeX>``: Elasticsearch IP of the node ``X``
        - ``<kibana_ip>``: Kibana server IP
        - ``<wazuh_master_server_IP>``: Wazuh Server IP

      You can add as many Elasticsearch nodes as needed. To generate certificates for them, the ``opendistro_security.nodes_dn`` must be also updated, adding the information of the new certificates. There must be the same number of certificates rows as nodes are on the installation.

      If you have more than one Wazuh server, you can add as many nodes as needed and certificates are created for each of them. To do this, change the ``name`` of the certificate and the ``CN`` value in the ``Clients certificates`` section: 

      .. code-block:: yaml

        - name: filebeat-X
          dn: CN=filebeat-x,OU=Docu,O=Wazuh,L=California,C=US                

>     - To install Elasticsearch, run the script with the option ``-e``, ``-c``, and ``-n <node_name>``. The name of the node must be the same used in ``config.yml`` for the certificate creation, e.g. ``master_node_1``:

Reviewer: ```suggestion
    - To install Elasticsearch, run the script with the options ``-e``, ``-c``, and ``-n <node_name>``. The name of the node must be the same used in ``config.yml`` for the certificate creation, e.g. ``master_node_1``:
```

        # curl -so ~/elastic-stack-installation.sh https://raw.githubusercontent.com/wazuh/wazuh-documentation/|WAZUH_LATEST_MINOR|/resources/open-distro/unattended-installation/distributed/elastic-stack-installation.sh 

>     - In order to install the subsequent nodes, run the script with the option ``-e`` and ``-n <node_name>``. The name of the node must be the same used in ``config.yml`` for the certificate creation, e.g. ``master_node_x``:

Reviewer: ```suggestion
    - In order to install the subsequent nodes, run the script with the options ``-e`` and ``-n <node_name>``. The name of the node must be the same used in ``config.yml`` for the certificate creation, e.g. ``master_node_x``:
```

      .. code-block:: console

        # /usr/share/elasticsearch/plugins/opendistro_security/tools/securityadmin.sh -cd /usr/share/elasticsearch/plugins/opendistro_security/securityconfig/ -icl -nhnv -cacert /etc/elasticsearch/certs/root-ca.pem -cert /etc/elasticsearch/certs/admin.pem -key /etc/elasticsearch/certs/admin.key -h <elasticsearch_IP>
           
    

.. note:: The Open Distro for Elasticsearch performance analyzer plugin is removed during the installation to prevent any negative impact on system resources. 

Configuring Elasticsearch
^^^^^^^^^^^^^^^^^^^^^^^^^

Once Elasticsearch is installed, the script starts the services automatically. The certificates are placed at ``~/certs.tar``. This file must be copied into the :ref:`Wazuh server <unattended_distributed_wazuh>` to extract the certificates needed.

In case Kibana is installed on a different server, the ``certs.tar`` file should be also copied into the Wazuh server to extract the corresponding certificates.

.. _install_kibana_unattended:

Installing Kibana
-----------------

> #. Download the script. Skip this step if you have Kibana installed on the same server as Open Distro for Elasticsearch:

Reviewer: ```suggestion
#. Download the script. Skip this step if you are installing Kibana on the same server as Open Distro for Elasticsearch:
```
